contrib/mac-build

```diff
@@ -30,11 +31,15 @@ function reporterror {
 
 echo =============================================================================
 echo
-echo "running: sudo port install $macports_prereq"
 if which port &>/dev/null; then
+  echo "running: sudo port install $macports_prereq"
   echo "You may be asked for your password to install these packages."
   echo
   sudo port install $macports_prereq || reporterror
+elif which brew &>/dev/null; then
+  for package in $homebrew_prereq; do
+      brew list $package &>/dev/null || brew install $package || reporterror
```

Comment:
Here is the full output:
https://gist.github.com/pathawks/98a254136d135ad5b9285ce1350c4d90

It is not currently interactive. Is that something that should be added?